### PR TITLE
Improvements and fixes for the docker development experience

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
Goes hand-in-hand with the corresponding `clintb/development-build-fixes` branch in these repos:
https://github.com/tidepool-org/blip
https://github.com/tidepool-org/viz

See this trello card for any additional details:
https://trello.com/c/PnIamSBO

We will need the corresponding `viz` PR approved first, and a docker image pushed for it before we can merge this to master.

### CODE REVIEW

**Comment by cbwebdevelopment** [comment]
[Improvements and fixes for the docker development experience](https://trello.com/c/PnIamSBO/228-improvements-and-fixes-for-the-docker-development-experience)

File: docker-compose.yml [hunk 1]
```diff
@@ -128,12 +128,25 @@ services:
...
       - '8078:8078'
 
+  # viz:
+  #   image: viz
+  #   container_name: viz
+  #   volumes:
+  #     - $TP_VIZ_DIR:/app:cached
+  #   environment:
+  #     NODE_ENV: development
+  #   expose:
```
Comment (pazaan) [inline docker-compose.yml]: `expose` isn't required if viz isn't needed by any other service. The user can already get to the port via the `ports` directive.

Comment (cbwebdevelopment) [inline docker-compose.yml]: Right.  I'll remove it here, and from `blip` service as well.

File: README.md [hunk 2]
```diff
@@ -138,10 +139,56 @@ To go back to running the container with `DEV_TOOLS` set to true:
...
+  - `tidepool link blip tideline`
+  - `tidepool unlink blip viz`
+- run npm (yarn) scripts in a service
+  - `tidepool blip install`
+  - `tidepool tideline run test-watch`
+  - `tidepool viz run stories`
+
+This script will only work in a Linux or MacOS environment (though Windows users may be able to get it working in [GitBash](https://git-for-windows.github.io/) or the new [Bash integration in Windows 10](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide))
+
+While you can run the script in this directory via `./tidepool`, it's recommended that you copy it over to a location where it will be found in your `$PATH`, such as your `usr/local/bin` directory.
```
Comment (cbwebdevelopment) [inline README.md]: Thanks, I'll fix this.

**Pull request merged**